pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.fr-fr.md

```diff
@@ -6,84 +6,219 @@ updated: 2024-04-16
+
+La version utilisé pour l'ensemble de nos produit Vcloud Director (VCD) est la `version 1.5`
 
 ## Prérequis
 
 >[!primary]
 > Si vous ne savez comment vous connecter au portail web de votre organisation, consultez d'abord [ce guide](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-logging).
 
-- Avoir un compte VMware Cloud Director
+- Une connaissance pratique des réseaux Linux, vSphere, Windows et IP est nécessaire pour configurer et gérer vCD
+- Avoir un compte VMware Cloud Director administrateur
+- Avoir une infrastructure VMware Cloud Foundation (VCF)
+  - vSphere et des ressources NSX (selon offre)
+  - VSan (selon offre)
 
 ## En pratique
 
-VMware vCloud Director (VCD) est une plateforme de gestion de cloud computing. Celle-ci permet la création, la gestion et le déploiement de ressources informatiques virtualisées à grande échelle. VCD offre une infrastructure agile et évolutive. Grâce à son interface utilisateur conviviale et à ses fonctionnalités avancées telles que la gestion des ressources, la facturation automatisée et la sécurité renforcée, vCloud Director simplifie la gestion des environnements cloud complexes.
+VMware vCloud Director (VCD) est une plateforme de gestion de cloud computing. Celle-ci permet la création, la gestion et le déploiement de ressources informatiques virtualisées à grande échelle. VCD offre une infrastructure agile et évolutive. Grâce à son interface utilisateur conviviale et à ses fonctionnalités avancées telles que la gestion des ressources, du réseau logiciel, et la sécurité renforcée, vCloud Director simplifie la gestion des environnements cloud complexes.
 
-Cette solution vous permet de provisionner et de gérer efficacement des machines virtuelles, des réseaux virtuels et d'autres ressources, offrant ainsi une agilité opérationnelle et une flexibilité accrues pour répondre aux besoins changeants des entreprises.
+[comment]: <> (Réécriture de l'introduction "En pratique" pour accès plus le propos sur le service que fournis OVH afin de se démarquer de VMware)
+VMware vCloud Director (VCD) permet d'avoir un service reuni au sein d'une même organisation managé par OVHcloud.
 
 >[!primary]
-> Une fois connecté à votre interface web, vous serez accueilli par un tableau de bord affichant vos **vDC**, ainsi qu'un résumé détaillé de votre utilisation des ressources (10). En haut de l'écran, vous trouverez également une barre de navigation regroupant les différentes options de paramétrage disponibles pour vCloud Director (VCD).
+> Une fois connecté à votre interface web, vous serez accueillis par un tableau de bord affichant vos **vDC**, ainsi qu'un résumé détaillé de votre utilisation des ressources (10). En haut de l'écran, vous trouverez également une barre de navigation regroupant les différentes options de paramétrage disponibles pour vCloud Director (VCD).
 
 ![Dashboard Overview](images/vcd-dashboard-overview.png){.thumbnail}
 
-Les différentes sections disponibles :
+<details>
```

Comment:
@pg-ovh , I suggest you remove all the `<details>` formats from this guide as they are not necessary.

If they were to be kept, they would need fixes as there are too many `<summary>` tags and all included content has to be formatted in HTML too, as in https://help.ovhcloud.com/csm/fr-vmware-nsx-interface-user-access?id=kb_article_view&sysparm_article=KB0062999 (see formats in https://github.com/ovh/docs/blob/develop/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/nsx_add_user_rights/guide.fr-fr.md?plain=1)

Reply:
@OVH-Team-Guides also from the en-gb guide ?

Reply:
Yes please